### PR TITLE
enh(elixir) improve heredoc and sigil support

### PR DESCRIPTION
**Pull request by yyyc514**

### CODE REVIEW

**Comment by yyyc514** [comment]
My attempt to save *something* from #927.

Turned out just to be a markup test that I then went on to expand.

I believe this solves points 1 and 2 from #730

File: src/languages/elixir.js [hunk 1]
```diff
@@ -19,16 +19,78 @@ function(hljs) {
...
     variants: [
+      {
+        begin: /"""/, end: /"""/,
+      },
+      {
+        begin: /'''/, end: /'''/,
+      },
+      {
+        begin: /~S"""/, end: /"""/,
+        contains: []
```
Comment (egor-rogov) [inline src/languages/elixir.js]: I don't get... What is the point in empty `contains`?

Comment (yyyc514) [reply src/languages/elixir.js]: It's needed to override the contains on line 64... variants "inherit", so it's needed to break that.

Comment (egor-rogov) [reply src/languages/elixir.js]: Ah.

**Pull request merged**